### PR TITLE
Add background

### PR DESCRIPTION
**Pull request by VivienneSoldier**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -6,3 +6,10 @@
   color: #2c3e50;
   margin-top: 60px;
 }
+body {
+  background-image: url(src/image/bg-image.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 100vh;
+  min-width: 100wh;
```
Comment (lagdush) [inline style.css]: Here's a typo.

**Review by lagdush — CHANGES_REQUESTED** [review]

File: style.css [hunk 2]
```diff
@@ -6,3 +6,10 @@
   color: #2c3e50;
   margin-top: 60px;
```
Comment (lagdush) [inline style.css:7]: Do we need margin here?

**Comment by lagdush** [comment]
Please delete testComponent

**Comment by lagdush** [comment]
the background should be centered on each screen
Zrzut ekranu 2021-12-1 o 21 33 29

**Comment by lagdush** [comment]
the screen shows scroll unnecessarily. Why do you think, could you fix this?

**Comment by VivienneSoldier** [comment]
> the screen shows scroll unnecessarily. Why do you think, could you fix this?

I think it should be easy to fix it. But now i'm sick, and it's getting worse. I need a little time to cure and to do more with all of this. Its ok to be a bit slower than other (and to talk about this right here xd)?

File: style.css [hunk 3]
```diff
@@ -6,3 +6,22 @@
...
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center, center;
+  background-attachment: fixed
+}
+@media only screen and (max-width: 375px) {
+body {
+  background-size: 580px 660px;
+}
+}
```
Comment (lagdush) [inline style.css]: The background on mobile screens is distorted.

File: style.css [hunk 4]
```diff
@@ -6,3 +6,22 @@
...
 }
+* {
+  margin:0;
+  padding:0;
+  box-sizing: border-box;
+}
+body {
+  background-image: url(src/image/bg-image.jpg);
+  width: 100vw;
+  height: 100vh;
```
Comment (lagdush) [inline style.css]: try to delete these values.

**Review by lagdush — CHANGES_REQUESTED** [review]
Delete commented lines of code.

**Review by lagdush — APPROVED** [review]

**Pull request merged**